kolena/workflow/model.py

```diff
@@ -159,6 +159,33 @@ def load(cls, name: str, infer: Optional[Callable[[TestSample], Inference]] = No
         log.info(f"loaded model '{name}' ({get_model_url(obj._id)})")
         return obj
 
+    @classmethod
+    @with_event(event_name=EventAPI.Event.LOAD_ALL_MODEL)
+    def load_all(
+        cls,
+        *,
+        infer: Optional[Callable[[TestSample], Inference]] = None,
```

Comment:
Do we need this argument?

Reply:
I asked myself the same thing. We have it for `load` so I added it here as well for parity. A valid use for a mutli-model inference function be one which uses model name as part of mapping to a stored precomputed inference.

Reply:
Is this blocking this PR?

Reply:
The `infer` is a function that takes a sample and output an inference, the example you mentioned doesn't exactly fit. A single infer for multiple models does not make sense to me.

For parity, I'd say we should provide a list of infer functions, but that conflicts with `load_all`, as users likely wouldn't know.

I think we can go without  this arguments, and users should be able to use the results from this endpoint to `reinitialize` a model instance with the right `infer`.

Reply:
> I think we can go without this arguments, and users should be able to use the results from this endpoint to reinitialize a model instance with the right `infer`.

This is a pretty bad devx, but I agree that it's better than applying one `infer` function to potentially many models returned from `load_all`.

```python3
models = Model.load_all(tags=["example"])
model = Model.load(models[0].name, infer=infer) if models else Model.create(new_name, infer=infer)
```